ODIN_II/SRC/verilog_bison.y

```diff
@@ -721,6 +723,7 @@ expression:
 	| '+' expression %prec UADD						{$$ = newUnaryOperation(ADD, $2, my_yylineno);}
 	| '-' expression %prec UMINUS					{$$ = newUnaryOperation(MINUS, $2, my_yylineno);}
 	| '~' expression %prec UNOT						{$$ = newUnaryOperation(BITWISE_NOT, $2, my_yylineno);}
+    | vBUF expression %prec UBUF					{$$ = newUnaryOperation(BUF_NODE, $2, my_yylineno);}
```

Comment:
remove this since buf is a gate it needs to be instantiated with the full name and not an operator
```
buf(...)
```
but never 
```
out = buf(a)
```

buf is never part of an assignment